sensor/upgrader/plan/preserve_test.go

```diff
@@ -152,7 +153,7 @@ func TestPreserveResources(t *testing.T) {
 	var mergedDS v1.DaemonSet
 	require.NoError(t, convert(scheme.Scheme, newDSUnstructured, &mergedDS))
 
-	assert.Equal(t, expectedMergedDS, &mergedDS)
+	protoassert.Equal(t, expectedMergedDS, &mergedDS)
```

Comment:
This type is a k8s.io type, none of our generated types (as far as I can tell). It matches the `proto.Message` interface, though.